presto-docs/src/main/sphinx/admin/verifier.rst

```diff
@@ -286,6 +286,7 @@ Name                                        Description
 ``running-mode``                            Set to ``query-bank`` to make the Verifier run in ``query-bank`` mode. Supports
                                             ``query-bank`` and ``control-test``. Defaults to ``control-test``.
 ``save-snapshot``                           Set to ``true`` to save checksums to ``mysql`` database.
+``insert-extended-verification``            Set to ``true`` to run extended verification for INSERT queries.
```

Comment:
It can also apply to ctas in addition to insert, right? In that case you can have a more general naming of the config.